fgpyo/io/__init__.py

```diff
@@ -0,0 +1,200 @@
+"""
+fio
+-------
+Module for reading and writing files.
```

Comment:
```suggestion
Module for reading and writing files
------------------------------------
```

Title should be helpful text; sphinx will usually fail if the `---` isn't the same length as the title (sigh).